apps/api-runtime/src/main/java/com/akto/utils/DataInsertionUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.akto.utils;
+
+import java.util.List;
+
+import com.akto.dao.AccountSettingsDao;
+import com.akto.dto.AccountSettings;
+import com.akto.dto.HttpResponseParams;
+import com.akto.parsers.HttpCallParser;
+import com.akto.runtime.APICatalogSync;
+
+public class DataInsertionUtil {
+
+    public static void processTraffic(HttpCallParser httpCallParser, List<HttpResponseParams> responses, int apiCollectionId){
```

Comment:
Use the same function in https://github.com/akto-api-security/akto/blob/master/apps/dashboard/src/main/java/com/akto/utils/Utils.java#L420

Reply:
skippng this now, will modify this as a part of different pr